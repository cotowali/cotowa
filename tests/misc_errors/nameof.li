nameof('10')
nameof(1 + 1)
var a: int
nameof(a + a)
