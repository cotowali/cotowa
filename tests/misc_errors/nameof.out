error: nameof.li 1,8: expression does not have a name
    1 | nameof('10')
      |        ^^^^
error: nameof.li 2,8: expression does not have a name
    2 | nameof(1 + 1)
      |        ^^^^^
error: nameof.li 4,8: expression does not have a name
    4 | nameof(a + a)
      |        ^^^^^
